stream-chat-android-sample/src/main/kotlin/io/getstream/chat/sample/application/App.kt

```diff
@@ -30,6 +37,19 @@ class App : Application() {
                 )
             notificationHandler = SampleNotificationHandler(this@App, notificationConfig)
         }.build()
+
+        configFlipper()
```

Comment:
It's good to split App for debug and other build types and make flipper a debug dependency

Reply:
I thought it wouldn't be a problem since this is just the sample app. Splitting the Application into Debug and Release version will work, but it is simpler to use the no-op version of Flipper for release. Like it is suggested: https://fbflipper.com/docs/getting-started/android-native. Just for the sample app, of course, we don't want to send this dependency together with our SDKs.

Reply:
If I remember correctly, we'll run into the issue that the main Flipper library has a no-op version, but the network interceptor add-on - and others we might include - doesn't. So we'll end up having to go the same route as we have now with Stetho, with an initializer that has different implementations for release/debug variants.

Reply:
No-op includes the add ons, so I don't think we would have this sort of problem. I added a commit that shows that: `adding network add-on for flipper`.

I'm not really advocating that we use the no-op, both solutions are fine.

Reply:
We can have a `ExtraDependecies` class that set up this configuration and have different implementations for Debug and Release. Then we don't need to do any `BuildConfig.DEBUG` check